### ISSUE TITLE
error when registering a new user on local

### ISSUE THREAD

**Issue by crystal-chan** [pantsel/konga#473]
i did a local installation and after i try to register a new user

i get this error

```
{
  "invalidAttributes": {
    "password": [
      {
        "rule": "minLength",
        "message": "\"minLength\" validation rule failed for input: 'admin'\nSpecifically, it threw an error.  Details:\n undefined"
      }
    ]
  },
  "model": "Passport",
  "_e": {},
  "rawStack": "    at WLValidationError.WLError (/Users/cchan/konga/node_modules/waterline/lib/waterline/error/WLError.js:25:15)\n    at new WLValidationError (/Users/cchan/konga/node_modules/waterline/lib/waterline/error/WLValidationError.js:19:28)\n    at _afterValidating (/Users/cchan/konga/node_modules/waterline/lib/waterline/query/validate.js:53:23)\n    at allValidationsChecked (/Users/cchan/konga/node_modules/waterline/lib/waterline/core/validations.js:265:12)\n    at /Users/cchan/konga/node_modules/waterline/node_modules/async/lib/async.js:52:16\n    at Object.async.forEachOf.async.eachOf (/Users/cchan/konga/node_modules/waterline/node_modules/async/lib/async.js:236:30)\n    at Object.async.forEach.async.each (/Users/cchan/konga/node_modules/waterline/node_modules/async/lib/async.js:209:22)\n    at module.exports.Validator.validate (/Users/cchan/konga/node_modules/waterline/lib/waterline/core/validations.js:144:9)\n    at /Users/cchan/konga/node_modules/waterline/lib/waterline/query/validate.js:42:25\n    at /Users/cchan/konga/node_modules/waterline/node_modules/async/lib/async.js:718:13\n    at Immediate.iterate (/Users/cchan/konga/node_modules/waterline/node_modules/async/lib/async.js:262:13)\n    at runCallback (timers.js:810:20)\n    at tryOnImmediate (timers.js:768:5)\n    at processImmediate [as _immediateCallback] (timers.js:745:5)",
  "reason": "1 attribute is invalid",
  "code": "E_VALIDATION",
  "status": 400,
  "details": "Invalid attributes sent to Passport:\n • password\n   • \"minLength\" validation rule failed for input: 'admin'\nSpecifically, it threw an error.  Details:\n undefined\n",
  "message": "[Error (E_VALIDATION) 1 attribute is invalid] Invalid attributes sent to Passport:\n • password\n   • \"minLength\" validation rule failed for input: 'admin'\nSpecifically, it threw an error.  Details:\n undefined\n",
  "stack": "Error (E_VALIDATION) :: 1 attribute is invalid\n    at WLValidationError.WLError (/Users/cchan/konga/node_modules/waterline/lib/waterline/error/WLError.js:25:15)\n    at new WLValidationError (/Users/cchan/konga/node_modules/waterline/lib/waterline/error/WLValidationError.js:19:28)\n    at _afterValidating (/Users/cchan/konga/node_modules/waterline/lib/waterline/query/validate.js:53:23)\n    at allValidationsChecked (/Users/cchan/konga/node_modules/waterline/lib/waterline/core/validations.js:265:12)\n    at /Users/cchan/konga/node_modules/waterline/node_modules/async/lib/async.js:52:16\n    at Object.async.forEachOf.async.eachOf (/Users/cchan/konga/node_modules/waterline/node_modules/async/lib/async.js:236:30)\n    at Object.async.forEach.async.each (/Users/cchan/konga/node_modules/waterline/node_modules/async/lib/async.js:209:22)\n    at module.exports.Validator.validate (/Users/cchan/konga/node_modules/waterline/lib/waterline/core/validations.js:144:9)\n    at /Users/cchan/konga/node_modules/waterline/lib/waterline/query/validate.js:42:25\n    at /Users/cchan/konga/node_modules/waterline/node_modules/async/lib/async.js:718:13\n    at Immediate.iterate (/Users/cchan/konga/node_modules/waterline/node_modules/async/lib/async.js:262:13)\n    at runCallback (timers.js:810:20)\n    at tryOnImmediate (timers.js:768:5)\n    at processImmediate [as _immediateCallback] (timers.js:745:5)"
}
```

**Comment by crystal-chan** [comment]
im assuming this means that the password is too short? but there is no indication of that on the FE. 
when i check the db though, the user is created

**Comment by pantsel** [comment]
@crystal-chan ,

thanks for the feedback.

The issue is fixed on [v0.14.5]( https://github.com/pantsel/konga/releases/tag/0.14.5)

Cheers
